### PR TITLE
Add list of contributors

### PR DESCRIPTION
**Pull request by jxtx**

### CODE REVIEW

File: LICENSE.txt [hunk 1]
```diff
@@ -13,7 +13,7 @@ Licensed under the Academic Free License version 3.0
        Work, thereby creating derivative works ("Derivative Works") based upon 
        the Original Work;
 
-    c) to distribute or communicate copies of the Original Work and Derivative 
+    c) to distribute or communicate copies of the Original Work aGnd Derivative
```
Comment (dannon) [inline LICENSE.txt]: s/aGnd/and

**Comment by nekrut** [comment]
+1, but I will need to add Tobacco Funds and Huck Funds. I will issue a separate PR for this

**Comment by natefoo** [comment]
Many of these email addresses are now invalid, not sure if we should include them at all?

**Comment by jxtx** [comment]
I like including them since they are used in the commit messages. Happy for people to update with their current address but I'd still like to keep the old ones.

**Comment by jmchilton** [comment]
I think this pull request should probably follow the procedures laid out for governance changes introduced in #314 - though I am absolutely fine if it doesn't - we made the process very onerous.

**Comment by natefoo** [comment]
@jmchilton Is this a governance change? Just recognition for past committers, right?

**Comment by jmchilton** [comment]
We are clarifying the license also - #314 mentions clarifications specifically.

**Comment by jxtx** [comment]
I think #314 only mentions clarification of procedures. No procedure changes here. I would like for #314 not to apply for this one.

**Comment by jmchilton** [comment]
@jxtx No objections here then :).

**Comment by dannon** [comment]
+1, and I also don't think this is a procedural change.

**Comment by jmchilton** [comment]
I am -0 on the current "Institutional sponsors" language - feel free to merge without winning me over on it obviously. No need to let me bite the hand that feeds me :).

More specifically, I don't mind the idea of calling out specific grants that have funded Galaxy core development explicitly, but by introducing it via the institutions it makes it sound for instance as though more Galaxy core development has occurred by employees of say George Washington University _while they were employed there_ than employees of the University of Freiburg _while they were employed there_.

My preference would be something more like the following - which still calls out the same four institutions for their leadership role - but which also makes a more clear distinction I believe between leadership and contribution:

---

# History

Galaxy development began at The Pennsylvania State University in 2006. In 2009 all contributions to that point were licensed by The Pennsylvania State University under the terms of the Academic Free License 3.0 (see LICENSE.txt). This license applies to all subsequent contributions as well. Galaxy development has continued in part due the grants and institutions listed below.

# Grant Sponsorship

Development of the core Galaxy project has been explicitly funded by the following grants.
- NSF DBI 0543285, “Tailoring genomic data to the needs of experimental biologists and educators”
- NIH R21 HG005133, “A turnkey solution for next generation sequence data analysis”
- NIH R01 HG004909, “An efficient lightweight environment for biomedical computation”
- NSF DBI 0850103, “Cyberinfrastructure for accessible and reproducible research in life sciences”
- NIH RC2 HG005542, “Dynamically scalable accessible analysis for next generation sequence data”
- NIH U41 HG006620, “Democratization of data analysis in life sciences through Galaxy”

# Institutional Sponsorship

The following institutions have significantly contributed to the development of the core Galaxy project (those marked with \* through the grants explicitly outlined above).
- The Pennsylvania State University*
- Emory University*
- Johns Hopkins University*
- George Washington University*
- University of Freiburg
- University of Minnesota
- Texas A&M University

**Comment by martenson** [comment]
+1 @jmchilton's approach
Should the institutions list correspond to the commiters group?

**Comment by natefoo** [comment]
At least in the case of Penn State, there were funds (and not just salary funds) contributed, so perhaps that's what @jxtx was driving at in that section?

**Comment by jmchilton** [comment]
I go from -0 to +0 for the language on institutional support if the language is modified corresponding to https://github.com/jxtx/galaxy/pull/5.

So I am :+1: on the pull request now.

**Comment by dannon** [comment]
Still +1.

**Comment by nekrut** [comment]
+1

**Comment by bgruening** [comment]
If I'm counting correctly we have more than enough votes to merge this.
Speaking as someone from the community I think we all appreciate this PR (and the coming changes) very much!

Thanks for moving the entire project in this direction. ❤️

**Comment by mr-c** [comment]
FYI, here is how I harvested the list of contributors from Git including name & email address cleanup using the `.mailmap` feature of `git log` https://github.com/dib-lab/khmer/pull/1023

In this case I'm outputting LaTeX for our software paper; should be easy to adjust for your own needs.

**Pull request merged**